src/proxy/cache/proxy.js

```diff
@@ -511,53 +519,58 @@ export default _.defaults(
                 }
             };
 
+            // the proxy options are supplied after the 'init' phase, we need to apply them later
+            this.itemStore.setItemTTL(this.getItemTTL());
```

Comment:
I don't understand why config options wouldn't be available at `install` or `init` (they seem to be passed)? It looks weird to set again the store TTL every time `getItem` is called.

Reply:
I know, this is weird. I spend some time trying to workaround this, but in fact, the config is given as the result of the `init` action, called by the proxy init. See https://github.com/oat-sa/tao-test-runner-qti-fe/blob/7bfb016066cf838629b6e7703e340037bbaeafd6/src/provider/qti.js#L613

A refactoring would be needed to supply these values at a more appropriate time, but it is beyond this task.